### PR TITLE
S8497: Adding entries to open layers build as part of user defined wfs connections work. 

### PR DESCRIPTION
**Pull request by ranjithguru**
S8497: Adding entries to open layers build as part of user defined wfs connections work.

### CODE REVIEW

File: build/di.cfg [hunk 1]
```diff
@@ -30,6 +30,11 @@ OpenLayers/Format/CQL.js
 OpenLayers/Format/GeoJSON.js
 OpenLayers/Format/XML.js
 OpenLayers/Format/WMSCapabilities/v1_1_1.js
+OpenLayers/Format/WFSCapabilities.js
+OpenLayers/Format/WFSCapabilities/v1.js
+OpenLayers/Format/WFSCapabilities/v1_0_0.js
+OpenLayers/Format/WFSCapabilities/v1_1_0.js
+OpenLayers/Format/WFSDescribeFeatureType.js
```
Comment (ziroby) [inline build/di.cfg:34]: Why do we have to include multiple versions of `WFSCapabilities`?  (I''m sorry, I don't understand this config file format)

**Comment by ranjithguru** [comment]
Hi ziroby,
                I am using WFS capability version v1.1.0 in my code I think we can remove lines,
OpenLayers/Format/WFSCapabilities/v1.js
OpenLayers/Format/WFSCapabilities/v1_0_0.js

Please suggest me whether adding   OpenLayers/Format/WFSCapabilities/v1_1_0.js is enough ? or just OpenLayers/Format/WFSCapabilities.js will work as it defaults to version 1.1.0 ? This is the first time I am doing this Please suggest.

**Comment by ranjithguru** [comment]
I have added only the required entries please review.

**Comment by ziroby** [comment]
LGTM

**Pull request merged**